pyproject.toml

```diff
@@ -116,6 +117,7 @@ doc = [
     "pydata-sphinx-theme", # Required for building documentation
     "sphinx",              # Required for building documentation, v7.2 errors
     "sphinx-design",       # Required for building documentation
+    "sphinxcontrib-autoprogram", # Required for adding command line options to documentation
```

Comment:
Oooh, is this all that is needed to auto-document command line options?  Great.

Reply:
Yep, this will work for the CLI too. I think we'd need to play with it a bit to get it to format nicely for all of the subcommands, but it is supposed to be able to do it.